apis/api-journeys-modern/src/schema/aiTranslateJourney/aiTranslateJourney.ts

```diff
@@ -0,0 +1,40 @@
+import { Job, Queue } from 'bullmq'
+
+import { connection } from '../../lib/redisConnection'
+import { builder } from '../builder'
+
+const queueName = 'jfp-ai-translate-journey'
+const queue = new Queue(queueName, { connection })
+
+interface AiTranslateJourneyJob {
+  userId: string
+  journeyId: string
+  name: string
+  textLanguage: string
+  videoLanguage: string
+}
+
+builder.mutationField('aiTranslateJourneyCreate', (t) =>
```

Comment:
```suggestion
builder.mutationField('journeyTranslateCreate', (t) =>
```

something like this, @tataihono mentioned would should try to namespace well. So keeping all Journey related mutations together.